packages/notices/src/store/actions.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * External dependencies
+ */
+import { uniqueId } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { DEFAULT_CONTEXT } from './constants';
+
+/**
+ * Yields action objects used in signalling that a notice is to be created.
+ *
+ * @param {?string}                status                Notice status.
+ *                                                       Defaults to `info`.
+ * @param {string}                 content               Notice message.
+ * @param {?Object}                options               Notice options.
+ * @param {?string}                options.context       Context under which to
+ *                                                       group notice.
+ * @param {?string}                options.id            Identifier for notice.
+ *                                                       Automatically assigned
+ *                                                       if not specified.
+ * @param {?boolean}               options.isDismissible Whether the notice can
+ *                                                       be dismissed by user.
+ *                                                       Defaults to `true`.
+ * @param {?Array<WPNoticeAction>} options.actions       User actions to be
+ *                                                       presented with notice.
+ */
+export function* createNotice( status = 'info', content, options = {} ) {
```

Comment:
Do we have any docs about using generators with action creators? I would love to read it myself to better understand what is possible as of today. I feel like I'm missing some great capabilities by not knowing what options are provided out of the box.

Reply:
>Do we have any docs about using generators with action creators?

At the moment, I think they're best described by:

- https://github.com/WordPress/gutenberg/tree/master/packages/data#controls
- https://github.com/WordPress/gutenberg/tree/master/packages/redux-routine/

Reply:
Thanks for sharing thos links, I will check them out on Monday.